src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,27 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={cn('TodoInfo',
```

Comment:
it's a good practice to use object destructing
```suggestion
export const TodoInfo = ({ todo }) => {
  const {...} = todo
  
  return (
    <article className={cn('TodoInfo',
```